ios/IoReactNativeProximity.swift

```diff
@@ -42,22 +42,43 @@ class IoReactNativeProximity: RCTEventEmitter {
    Resolves to true or rejects if an error occurs.
     
    - Parameters:
+      - certificates: Array of base64 representing DER encoded X.509 certificate which are used to authenticate the verifier app
       - resolve: The promise to be resolved
       - reject: The promise to be rejected
   */
-  @objc(start:withRejecter:)
+  @objc(start:withResolver:withRejecter:)
   func start(
+    certificates: Array<Any>,
     _ resolve: @escaping RCTPromiseResolveBlock,
     reject: @escaping RCTPromiseRejectBlock
   ){
     do {
-      try Proximity.shared.start()
+      let certsData = parseCertificates(certificates)
+      try Proximity.shared.start([].isEmpty ? nil : [])
```

Comment:
Shouldn't there be `certsData` instead of an empty array? Because otherwise the `certsData` variable seems to be unused. Or was this line intentional?
```suggestion
      try Proximity.shared.start(certsData.isEmpty ? nil : certsData)
```

Reply:
Nice catch, I was testing something and I forgot to change it back. Thanks.